src/socketx.h

```diff
@@ -15,5 +16,6 @@ typedef int socklen_t;
 typedef int SOCKET;
 const int INVALID_SOCKET = -1;
 inline void closesocket(int x) { close(x); }
+#define GetSocketError()   errno
```

Comment:
This naming convention is inconsistent. Nothing else in the socket library uses CamelCase. Choose something consistent.

Reply:
Changed name to socket_error()